src/Atmos/Model/declare_prognostic_vars.jl

```diff
@@ -18,4 +18,4 @@ abstract type Precipitation <: PrognosticVariable end
 struct Rain <: Precipitation end
 struct Snow <: Precipitation end
 
-struct Tracers <: PrognosticVariable end
+struct Tracers{N} <: PrognosticVariable end
```

Comment:
Presumably `N` is the number of tracers? I'm not seeing it used though?

Reply:
Yes, should we add a doc string? It's not really being used, but it shows up in the tendency table.

Reply:
It can be added when it's used.